src/pages/index.tsx

```diff
@@ -3,6 +3,7 @@ import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 import GradeBook from '../GradeBook';
 import { SignIn, SignUp } from './accounts';
 import ConfirmSignUp from './accounts/ConfirmSignUp';
+import ForgotPassword from './accounts/ForgotPassword';
```

Comment:
We should do the import like what is done on line 4. (adding it to the list). You will probably need to import and re-export the component in the `./accounts/index.ts(x)` file. Refer to that file to see how the SignIn/SignUp were done.